### DISCUSSION TITLE
Update existing App

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#1814] [Help] [answered]
After the update I want to have all the new feautures 😃 but how?

**Comment by shogunpurple** [comment, marked as answer]
@Fabian1410 You may have to create a new app, or export/import an existing one

**Reply by Fabian1410** [reply]
okay :)

**Reply by Fabian1410** [reply]
After this the buttons are round :(

**Reply by Fabian1410** [reply]
![image](https://user-images.githubusercontent.com/65181345/123147595-85ff1300-d45f-11eb-98c6-168cb7b8d81e.png)

**Reply by Fabian1410** [reply]
before
![image](https://user-images.githubusercontent.com/65181345/123147630-91523e80-d45f-11eb-8998-a857429855d0.png)

**Reply by mike12345567** [reply]
Hi @Fabian1410 - this has happened as part of our move to use an opinionated theming system namely [Adobe's Spectrum](https://spectrum.adobe.com/) - this is what we use for the builder/portal itself.

This will come into its own in the future as we enable more theming options across all your Budibase apps - we hope that using one design system will make it much easier to build beautiful apps, quickly!

**Reply by Fabian1410** [reply]
okay, no problem :)

**Reply by Fabian1410** [reply]
Automation Webhooks shows in schematic and trigger url **http://myip:10000/undefined**

**Reply by mike12345567** [reply]
Hi @Fabian1410 - looks like there was an issue here, a fix will be released ASAP.
